src/patterns/gen-ai/aws-llama-index-data-loader/README.md

```diff
@@ -0,0 +1,134 @@
+from constructs import Construct
+from cdklabs.generative_ai_cdk_constructs import LlamaIndexDataLoader
+
+data_loader = LlamaIndexDataLoader(self, 'LlamaIndexDataLoader', )
+```
+
+## Initializer
+
+```
+new LlamaIndexDataLoader(scope: Construct, id: string, props: LlamaIndexDataLoaderProps)
+```
+
+Parameters
+
+- scope [Construct](https://docs.aws.amazon.com/cdk/api/v2/docs/constructs.Construct.html)
+- id string
+- props LlamaIndexDataLoaderProps
+
+## Pattern Construct Props
+
+
+| **Name**                               | **Type**                                                                                                                                               | **Required**                                              | **Description**                                                                                                                                                                                                                                                                                                                                                                                               |
+| :--------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| existingDashboard               | [aws_cloudwatch.Dashboard](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudwatch.Dashboard.html)                                | ![Optional](https://img.shields.io/badge/optional-4169E1) | Existing dashboard to be used by the construct. **Mutually exclusive** with ```dashboardName``` - only one should be specified.                                                                                                                                                                                                                                                           |
+| dashboardName | string | ![Optional](https://img.shields.io/badge/optional-4169E1) | A name for the dashboard which will be created. If not provided, the construct will create a new dashboard named 'AossMetricsDashboard'. **Mutually exclusive** with ```existingDashboard``` - only one should be specified.                                                                                                                                                                                                                                                            |
```

Comment:
pattern construct props don't match the construct

Reply:
Sorry had some local fixes that I needed to push 97a94356b0cd6484195e73760c92c740b30c57c9